.github/workflows/cmake.yml

```diff
@@ -0,0 +1,29 @@
+# Note: CMake support is community-based. The maintainers do not use CMake
+# internally.
+
+name: cmake
+
+on:
+    push:
+    pull_request:
+        branches:
+            - main
+
+permissions:
+    contents: read
+
+jobs:
+  cmake_min:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        config: ["Debug", "Release"]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: lukka/get-cmake@latest
+        with:
+          cmakeVersion: 3.13.0
```

Comment:
This is the main thing I'm trying to add. We added this in the Vulkan Validation Layers since we found it very easy for developers using newer CMake version to add code that didn't respect the minimum.